infra/scripts/docker_build_cross_aarch64_runtime.sh

```diff
@@ -22,6 +22,8 @@ else
 fi
 
 # docker image name
+# - for xenial, use DOCKER_IMAGE_NAME="nnfw/one-devtools" (default)
```

Comment:
as we may update default to bionic someday, this may not be correct. how about giving full name as
```
# - for xenial, use DOCKER_IMAGE_NAME="nnfw/one-devtools:xenial"
```
to make version neutral for this comment.

Reply:
Applied the suggestion :-D